File: 1421-find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.cpp
class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int count = 0;
        for(int i : nums){
            int countDigits = log10(i)+1;
            
            if((countDigits & 1) == 0)
                count++;
        }
        return count;
    }
};

File: 1421-find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int count = 0;
        for(int num : nums) {
            if(isEven(num))
                count++;
        }
        return count;
    }

    public boolean isEven(int num) {
        int digits = 0;
        while(num > 0) {
            digits++;
            num /= 10;
        }
        return digits%2 == 0;
    }
}